CHANGELOG.md

```diff
@@ -8,6 +8,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Added `WithPeerServiceName` option to sarama instrumentation. ([#119](https://github.com/signalfx/signalfx-go-tracing/pull/119))
+
+### Changed
+
+- Changed values for `SpanTypeMessageConsumer` and `SpanTypeMessageProducer` to `consumer` and `producer` respectively. ([#119](https://github.com/signalfx/signalfx-go-tracing/pull/119))
```

Comment:
I feel that this message is not really well describing what has been changed. I think this could be split into two e.g.:
- (in `Added` section) `PRODUCER` and `CONSUMER` span kinds.
- (in `Added` section as I guess assigning `queue` was a NOP, if I am not right then it should be in `Changed`) Sarama instrumentation assigns `PRODUCER` and `CONSUMER` span kinds.